README.md

```diff
@@ -1,63 +1,200 @@
 # UINavigationController & TableViews
 
-![ElonMusck](http://i.imgur.com/1i4lr5P.jpg?1)  
+![ElonMusk](http://i.imgur.com/1i4lr5P.jpg?1)  
 
 > When something is important enough, you do it even if the odds are not in your favor. ~[Elon Musk](https://en.wikipedia.org/wiki/Elon_Musk)
- 
 
-## Learning Objectives - The student should be able to..
+## Learning Objectives - The student should be able to...
 
 * Create content when we have a sense at what exactly was presented to the student here.
 
+## Introduction
 
-## What the student can do at this point 
+Remember those magic days of summer camp, carrying watermelons around and learning how to dance? In this lesson, you're going to build a simple app using a table view that lists all the songs in the _Dirty Dancing_ soundtrack so you can relive those glory days. When you're done, your app will look like this:
```

Comment:
LOL!!! I don't know why but this made me laugh really hard (even if it wasn't mean to be funny). I think it reminded me of Wet Hot American Summer. 

"Watermelons and dancing"